pkg/cloudprovider/cloud.go

```diff
@@ -82,18 +80,18 @@ type LoadBalancer interface {
 	// TODO: Break this up into different interfaces (LB, etc) when we have more than one type of service
 	// GetLoadBalancer returns whether the specified load balancer exists, and
 	// if so, what its status is.
```

Comment:
Make it clear in the comments that the implementing functions are *not* allowed to change the service. (Or, if they do, they should not expect those changes to be persisted.)